### ISSUE TITLE
Add a `title` prop to Footer social links

### ISSUE THREAD

**Issue by GomezIvann** [dxc-technology/halstack-react#1645]
**Is your feature request related to a problem? Please describe.**
After the ADA compliance issues (#1644) we realized that our Footer icons are not accessible due to the lack of text, both visual and in screen readers.

**Describe the solution you'd like**
Add a title property to the `socialLinks` objects that will give value to properties `title` and `aria-label`.
